src/kixi/collect/campaign/aggregate/dynamodb.clj

```diff
@@ -0,0 +1,68 @@
+(ns kixi.collect.campaign.aggregate.dynamodb
+  (:require [com.stuartsierra.component :as component]
+            [kixi.collect.dynamodb :as db]
+            [kixi.collect.campaign.aggregate :as agrc]
+            [kixi.collect.aggregate :as agr]
+            [clojure.spec.alpha :as s]
+            [kixi.comms :as comms]
+            [kixi.spec.conformers :as sc]
+            [taoensso.timbre :as log]
+            [kixi.spec :refer [alias]]))
+
+(alias 'cc 'kixi.collect.campaign)
+(alias 'event 'kixi.event)
+
+(defn primary-campaign-table
+  [profile]
+  (str profile "-kixi.collect-campaigns"))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(defmulti handle-event
+  (fn [_ e] ((juxt ::event/type ::event/version) e)))
+
+(defmethod handle-event
+  [:kixi.collect/collection-requested "1.0.0"]
+  [{:keys [client table]} _]
+  (reify
+    agr/IAggregateEventDataHandler
+    (get [this event])
```

Comment:
should this have more body?

Reply:
Nah because this event doesn't care if anything already exists in the db